CppCoreGuidelines.md

```diff
@@ -19688,7 +19688,7 @@ implicit memory management, and `locale` handling.
 
 If you need I/O performance, you can almost always do better than `printf()`.
 
-`gets()` `scanf()` using `s`, and `printf()` using `%s` are security hazards (vulnerable to buffer overflow and generally error-prone).
+`gets()`, `scanf()` using `%s`, and `printf()` using `%s` are security hazards (vulnerable to buffer overflow and generally error-prone).
 In C11, they are replaced by `gets_s()`, `scanf_s()`, and `printf_s()` as safer alternatives, but they are still not type safe.
```

Comment:
This statement isn't true at all. The "safer" alternatives are optional in C11 and not widely supported, they don't replace anything.

Reply:
And I'm not sure what's wrong with `printf()` using `%s`, does it mean `sprintf`?

Reply:
printf_s would check if a null pointer is passed to %s (and also disable %n, if that matters)